### PR TITLE
Replace leapp-pr-welcome-msg-bot with GitHub action

### PR DESCRIPTION
**Pull request by andywaltlova**
For working example see [my test repo](https://github.com/andywaltlova/actions_test_space/pull/14)

(Same [PR in leapp](https://github.com/oamg/leapp/pull/712))

### CODE REVIEW

File: .github/workflows/pr-welcome-msg.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# IMPORTANT NOTE
+# In this workflow should NOT be checkout action - because of security reasons.
```
Comment (Wecros) [inline .github/workflows/pr-welcome-msg.yml:2]: should -> there should

**Review by Wecros — CHANGES_REQUESTED** [review]
Otherwise looks good. Good job with linking the private test repo :+1: Will merge after the change.

**Comment by Wecros** [comment]
Hmm, not sure why the `e2e` tests are failing. But I would guess the issue is not related to this PR.

File: .github/workflows/pr-welcome-msg.yml (hunk 1 above)
Comment (andywaltlova) [reply .github/workflows/pr-welcome-msg.yml]: Fixed

**Review by Wecros — APPROVED** [review]

**Comment by pirat89** [comment]
leapp-ci build

**Review by Wecros — APPROVED** [review]

**Comment by fernflower** [comment]
leapp ci-build

**Review by jmikovic — APPROVED** [review]

**Pull request merged**